qcodes/dataset/data_set.py

```diff
@@ -936,6 +942,43 @@ def get_data_as_pandas_dataframe(self,
             dfs[name] = df
         return dfs
 
+    def write_data_to_gnuplot_file(self, single_file: bool = False,
+                                   npath: Optional[str] = None) -> None:
+        """
+        A function to export data as GNU plot format.
```

Comment:
I guess gnuplot is one word and not capitalized. Note that it does not actually have anything to do with the GNU project https://en.wikipedia.org/wiki/Gnuplot